### DISCUSSION TITLE
type.raw and type_map.raw don not correspond to dft output

### DISCUSSION THREAD

**Discussion by hdjani** [deepmodeling/deepmd-kit#4272] [Q&A]
from my dft outputs (from Abinit), I have 8 A atoms, 8 B atoms and 24 C atoms. My type.raw file showed 1111111122222222000000000...000. But type_map.raw shows C A B that means that he considered 8 C, 8 A and 24 B. what should I put on my input.json

**Comment by wanghan-iapcm** [comment]
The type_map means that C is type 0, A is type 1 and B is type 2.
